.github/workflows/release.yaml

```diff
@@ -0,0 +1,52 @@
+on:
+  push:
+    branches:
+      - main
+name: release-please
+jobs:
+  release-please-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - id: release-pr
+        uses: GoogleCloudPlatform/release-please-action@v2
+        with:
+          token: ${{ secrets.FORKING_TOKEN }}
+          release-type: node
+          fork: true
+          package-name: googleapis
+          command: release-pr
+      - id: label
+        uses: actions/github-script@v3    
+        with:
+            github-token: ${{secrets.GITHUB_TOKEN}}
+            script: |
+              const [owner, repo] = process.env.GITHUB_REPOSITORY.split('/');
+              const latestRelease = await github.issues.addLabels({
+                owner,
+                repo,
+                issue_number: ${{steps.release-please.outputs.pr}},
```

Comment:
Is the step called `release-pr`, not `release-please`?

Reply:
(I have no idea actually :) )

Reply:
No, good catch, I believe it should be `release-pr`.